packages/messaging/src/index.ts

```diff
@@ -29,6 +34,9 @@ export const sendToBackground: PlasmoMessaging.SendFx = (req) =>
  */
 export const sendToActiveContentScript: PlasmoMessaging.SendFx = (req) =>
   new Promise(async (resolve, reject) => {
+    if (!chrome?.tabs) {
+      throw new Error("chrome.tabs is not available")
```

Comment:
Since we're abstracting away this API, a more specific message might be "chrome.tabs is not available. tabs permission hasn't been set in package.json"

Reply:
The reason why I didn't include the latter part is bc the user might try to call this API in a non-extension context.